pkg/storage/oss/oss.go

```diff
@@ -99,6 +107,32 @@ func (d *driver) UpdateEffectiveConfig() error {
 	return nil
 }
 
+func fetchCredentialsIniFromSecret(secret *corev1.Secret) (auth.Credential, error) {
```

Comment:
I have reorganized this code github.com/openshift/cluster-api-provider-alibaba/pull/17